core/src/wallet/pool/WalletPool.cpp

```diff
@@ -141,12 +141,12 @@ namespace ledger {
             const std::shared_ptr<api::DatabaseBackend> &backend,
             const std::shared_ptr<api::DynamicObject> &configuration,
             const std::shared_ptr<api::PreferencesBackend> &externalPreferencesBackend,
-            const std::shared_ptr<api::PreferencesBackend> &internalPreferencesBackend
-        ) {
+            const std::shared_ptr<api::PreferencesBackend> &internalPreferencesBackend) {
             auto pool = std::shared_ptr<WalletPool>(new WalletPool(
                     name, password, httpClient, webSocketClient, pathResolver,
                     logPrinter, dispatcher, rng, backend, configuration,
-                    externalPreferencesBackend, internalPreferencesBackend
+                    externalPreferencesBackend,
+                    externalPreferencesBackend
```

Comment:
The `internalPreferencesBackend` isn't use anymore. If this change is done in purpose, can you remove the variable name to the method parameter list or put a comment to indicate - in both case - why such a variable is unused.

Reply:
😨 good catch, internal is actually the important one.